Citation.cff

```diff
@@ -0,0 +1,12 @@
+cff-version: 1.2.0
+message: "If you use this software, please refer to our manuscript as below."
```

Comment:
Do we really require users to cite only the article ainstead of the software?  
There is also the solution software-references-article, i.e.



# This CITATION.cff file was generated with Zotero.

cff-version: 1.2.0
message: >-
  If you use this software, please cite it using the metadata from this file.
title:  >-
  vivo-project/VIVO
abstract: VIVO is an extensible semantic web application for research discovery and showcasing scholarly work
type: software
license: BSD-3-Clause
url: https://github.com/vivo-project/VIVO
date-released: 2024-08-11

references:
  - title: >-
      VIVO: a system for research discovery
    abstract: |
      VIVO [Pronunciation: vee-voh] is member-supported, enterprise open source software and an
      ontology for representing scholarship. VIVO supports recording, editing, searching, browsing
      and visualizing scholarly activity. VIVO encourages research discovery, expert finding, network
      analysis and assessment of research impact. VIVO is easily extended to support additional
      domains of scholarly activity (Börner, Conlon, Corson-Rikert, & Ying Ding, 2012).
      VIVO uses an ontology to represent people, papers, grants, projects, datasets, resources, and
      other elements of research and scholarship as linked open data. The ontology can be used to
      create RDF that can be loaded into VIVO. VIVO RDF data is easily exported for use in other
      applications.
      VIVO includes Vitro (Project, 2019), a domain-free engine for managing linked open data,
      the JFact reasoner (“JFact DL Reasoner,” 2018), SolR (“Apache Solr -,” 2019) for search,
      SPARQL query (“SPARQL Query Language for RDF,” 2008), Jena as a triple store (“Apache
      Jena -,” 2011), supporting both TDB (“Apache Jena - Apache Jena - TDB,” 2019) and
      SDB (“Apache Jena - SDB - persistent triple stores using relational databases,” 2019) on
      MySQL (“MySQL,” 2019), uses D3 (Bostock, 2015) for visualizations, and provides multiple
      APIs, including Triple Pattern Fragments (Verborgh et al., 2016) for rapid remote access to
      specified data.
      Using VIVO, organizations can represent the activities and accomplishments of their scholars
      as linked open data, and share that data with others.
    type: journalArticle
    issn: 2475-9066
    issue: 39
    languages: 
      - en
    pages: 1182
    volume: 4
    authors: 
      - family-names: Conlon
        given-names: Michael
      - family-names: Woods
        given-names: Andrew
      - family-names: Triggs
        given-names: Graham
      - family-names: O'Flinn
        given-names: Ralph
      - family-names: Javed
        given-names: Muhammad
      - family-names: Blake
        given-names: Jim
      - family-names: Gross
        given-names: Benjamin
      - family-names: Ahmad
        given-names: Qazi Azim Ijaz
      - family-names: Ali
        given-names: Sabih
      - family-names: Barber
        given-names: Martin
      - family-names: Elsborg
        given-names: Don
      - family-names: Fofack
        given-names: Kitio
      - family-names: Hauschke
        given-names: Christian
      - family-names: Ilik
        given-names: Violeta
      - family-names: Khan
        given-names: Huda
      - family-names: Lawless
        given-names: Ted
      - family-names: Levernier
        given-names: Jacob
      - family-names: Lowe
        given-names: Brian
      - family-names: Martin
        given-names: Jose Luis
      - family-names: McKay
        given-names: Steve
      - family-names: Porter
        given-names: Simon
      - family-names: Walther
        given-names: Tatiana
      - family-names: White
        given-names: Marijane
      - family-names: Wolff
        given-names: Stefan
      - family-names: Younes
        given-names: Rebecca
    date-published: 2019-07-26
    doi: 10.21105/joss.01182